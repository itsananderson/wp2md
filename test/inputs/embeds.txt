=== Video Embeds ===

Basic test of various video embeds

== Description ==

= YouTube Videos =

[youtube http://www.youtube.com/watch?v=bW6Y_CBQJNI]

Before video [youtube http://www.youtube.com/watch?v=bW6Y_CBQJNI] after video

Just the URL:

http://www.youtube.com/watch?v=bW6Y_CBQJNI

http://youtube.com/watch?v=bW6Y_CBQJNI

https://www.youtube.com/watch?v=bW6Y_CBQJNI

https://youtube.com/watch?v=bW6Y_CBQJNI

Text before https://youtube.com/watch?v=bW6Y_CBQJNI text after

Text before https://youtube.com/watch?v=bW6Y_CBQJNI. With a period

= Vimeo Videos =

[vimeo http://vimeo.com/173714]

Before video [vimeo http://vimeo.com/173714] after video

http://vimeo.com/173714

https://vimeo.com/173714

https://www.vimeo.com/173714

Text before https://www.vimeo.com/173714 text after

Text before https://www.vimeo.com/173714. With a period

= VideoPress =

[wpvideo OO4thna8]

Before video [wpvideo OO4thna8] and after

Before video [wpvideo OO4thna8]. With a period