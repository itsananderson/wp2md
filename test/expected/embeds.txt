Video Embeds
============

Basic test of various video embeds

Description
-----------

#### YouTube Videos ####

<a href="http://www.youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>

Before video <a href="http://www.youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a> after video

Just the URL:

<a href="http://www.youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>

<a href="http://youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>

<a href="https://www.youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>

<a href="https://youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>

Text before <a href="https://youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a> text after

Text before <a href="https://youtube.com/watch?v=bW6Y_CBQJNI" target="_blank"><img src="https://img.youtube.com/vi/bW6Y_CBQJNI/0.jpg" alt="" width="240" height="180" /></a>. With a period

#### Vimeo Videos ####

<a href="http://vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a>

Before video <a href="http://vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a> after video

<a href="http://vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a>

<a href="https://vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a>

<a href="https://www.vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a>

Text before <a href="https://www.vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a> text after

Text before <a href="https://www.vimeo.com/173714" target="_blank"><img src="http://b.vimeocdn.com/ts/209/721/20972155_640.jpg" alt="" width="240" height="180" /></a>. With a period

#### VideoPress ####

<a href="http://videos.videopress.com/OO4thna8/videopress2-web2.mp4" target="_blank"><img src="http://videos.videopress.com/OO4thna8/videopress2-web2.original.jpg" alt="" width="240" height="180" /></a>

Before video <a href="http://videos.videopress.com/OO4thna8/videopress2-web2.mp4" target="_blank"><img src="http://videos.videopress.com/OO4thna8/videopress2-web2.original.jpg" alt="" width="240" height="180" /></a> and after

Before video <a href="http://videos.videopress.com/OO4thna8/videopress2-web2.mp4" target="_blank"><img src="http://videos.videopress.com/OO4thna8/videopress2-web2.original.jpg" alt="" width="240" height="180" /></a>. With a period